src/wizard/apple/ios.md

```diff
@@ -16,7 +20,33 @@ target 'YourApp' do
 end
 ```
 
-Afterwards run `pod install`.
+Afterwards run `pod install`. For more information visit the [docs](/platforms/apple/install/cocoapods/).
```

Comment:
I originally thought we should get rid of this link, since the page we're linking to doesn't have any additional info, but the other two pages we link to:
- https://docs.sentry.io/platforms/apple/install/swift-package-manager/
- https://docs.sentry.io/platforms/apple/install/carthage/

do have some extra info (not a ton though)... Leaning toward keeping this link for consistency.

Reply:
I think we should leave for consistency.